builtin/aws/ecs/platform.go

```diff
@@ -128,6 +128,13 @@ func (p *Platform) ConfigSet(config interface{}) error {
 		}
 	}
 
+	if c.HealthCheck != nil {
+		if c.HealthCheck.Timeout >= c.HealthCheck.Interval {
+			return status.Errorf(codes.FailedPrecondition, "the health check "+
+				"timeout must be less than the interval")
```

Comment:
We should include the interval and timeout in the error message!

Reply:
Added the doc fields to `Documentation`, but `make gen/website-mdx` didn't generate the updated .mdx files 🤔 still looking into it.

Reply:
If I remember right, if it's a new markdown file, you have to manually `@include` it in like: https://github.com/hashicorp/waypoint/blob/main/website/content/plugins/aws-ec2.mdx. Not sure if that's the issue

Reply:
The ECS platform plugin docs indeed are included. Even more weirdly, this is passing the CircleCI website-mdx check... maybe it's because all of the fields in the new struct are optional?